core/src/software/aws/toolkits/core/telemetry/Metric.kt

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.core.telemetry
+
+import software.aws.toolkits.core.utils.getLogger
+import java.time.Duration
+import java.time.Instant
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
```

Comment:
I'm not sure if the use of `ZonedDateTime` is the right choice here. Since we'll be storing everything as epoch time anyways, doesn't it make sense to work directly with the timestamps with `Instant`?

Reply:
I agree, changed to `Instant`.